[Building] warning: unused variable

````
[0/2] Re-checking globbed directories...
[53/655] Building CXX object src/util/CMakeFiles/util_esbmc.dir/string_constant.cpp.o
/home/lucas/ESBMC_Project/esbmc/src/util/string_constant.cpp: In destructor ‘{anonymous}::switch_locale::~switch_locale()’:
/home/lucas/ESBMC_Project/esbmc/src/util/string_constant.cpp:117:17: warning: unused variable ‘n’ [-Wunused-variable]
  117 |     const char *n = setlocale(cat, orig.c_str()); // restore locale
      |                 ^
[183/655] Building CXX object src/goto-programs/abstract-interpretation/CMakeFiles/abstract-interpretation.dir/interval_analysis.cpp.
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_analysis.cpp: In function ‘void dump_intervals(std::ostringstream&, const goto_functiont&, const ait<interval_domaint>&)’:
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_analysis.cpp:240:29: warning: unused variable ‘d’ [-Wunused-variable]
  240 |     const interval_domaint &d = interval_analysis[i_it];
      |                             ^
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_analysis.cpp:241:10: warning: variable ‘print_vars’ set but not used [-Wunused-but-set-variable]
  241 |     auto print_vars = [&out, &i_it](const auto &map) {
      |          ^~~~~~~~~~
[214/655] Building CXX object src/goto-programs/abstract-interpretation/CMakeFiles/abstract-interpretation.dir/interval_domain.cpp.o
In file included from /home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.h:8,
                 from /home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.cpp:4:
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_template.h: In instantiation of ‘static interval_templatet<T> interval_templatet<T>::bitnot(const interval_templatet<T>&) [with T = BigInt]’:
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.cpp:557:27:   required from ‘T interval_domaint::get_interval(const expr2tc&) const [with Interval = interval_templatet<BigInt>; expr2tc = irep_container<expr2t>]’
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.cpp:1416:71:   required from here
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_template.h:526:68: warning: unused parameter ‘w’ [-Wunused-parameter]
  526 |   static interval_templatet<T> bitnot(const interval_templatet<T> &w)
      |                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_template.h: In instantiation of ‘static interval_templatet<T> interval_templatet<T>::bitnot(const interval_templatet<T>&) [with T = boost::multiprecision::number<boost::multiprecision::backends::cpp_bin_float<100> >]’:
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.cpp:557:27:   required from ‘T interval_domaint::get_interval(const expr2tc&) const [with Interval = interval_templatet<boost::multiprecision::number<boost::multiprecision::backends::cpp_bin_float<100> > >; expr2tc = irep_container<expr2t>]’
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.cpp:264:27:   required from ‘void interval_domaint::apply_assignment(const expr2tc&, const expr2tc&, bool) [with Interval = interval_templatet<boost::multiprecision::number<boost::multiprecision::backends::cpp_bin_float<100> > >; expr2tc = irep_container<expr2t>]’
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_domain.cpp:1244:55:   required from here
/home/lucas/ESBMC_Project/esbmc/src/goto-programs/abstract-interpretation/interval_template.h:526:68: warning: unused parameter ‘w’ [-Wunused-parameter]
[216/655] Building CXX object src/solidity-frontend/CMakeFiles/solidityfrontend_stuff.dir/solidity_convert.cpp.o
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp: In member function ‘bool solidity_convertert::convert()’:
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp:66:8: warning: variable ‘found_contract_def’ set but not used [-Wunused-but-set-variable]
   66 |   bool found_contract_def = false;
      |        ^~~~~~~~~~~~~~~~~~
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp: In member function ‘bool solidity_convertert::get_statement(const json&, exprt&)’:
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp:1427:16: warning: unused variable ‘rs’ [-Wunused-variable]
 1427 |         size_t rs = rhs.operands().size();
      |                ^~
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp:1467:16: warning: unused variable ‘rs’ [-Wunused-variable]
 1467 |         size_t rs = to_struct_type(rhs.type()).components().size();
      |                ^~
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp: In member function ‘bool solidity_convertert::get_binary_operator_expr(const json&, exprt&)’:
/home/lucas/ESBMC_Project/esbmc/src/solidity-frontend/solidity_convert.cpp:3006:16: warning: unused variable ‘rs’ [-Wunused-variable]
 3006 |         size_t rs = to_struct_type(new_rhs.type()).components().size();
      |                ^~
````

--- a/src/esbmc/options.cpp
+++ b/src/esbmc/options.cpp
@@ -293,9 +293,7 @@ const struct group_opt_templ all_cmd_options[] = {
      "disable return value optimization to compute the stack size"}}},
 
   {"Incremental SMT",
-   {{"smt-during-symex",
-     NULL,
-     "enable incremental SMT solving"},
+   {{"smt-during-symex", NULL, "enable incremental SMT solving"},
     {"smt-thread-guard",
      NULL,
      "check the thread guard during thread exploration"},

--- a/src/goto-programs/abstract-interpretation/interval_analysis.cpp
+++ b/src/goto-programs/abstract-interpretation/interval_analysis.cpp
@@ -237,7 +237,6 @@ void dump_intervals(
 {
   forall_goto_program_instructions (i_it, goto_function.body)
   {
-    const interval_domaint &d = interval_analysis[i_it];
     auto print_vars = [&out, &i_it](const auto &map) {
       for (const auto &interval : map)
       {

